pages/common/mpc.md

```diff
@@ -0,0 +1,28 @@
+# mpc
+
+> Music Player Client.
+> Program for controlling Music Player Daemon (MPD).
+
+- Toggle between play and pause:
+
+`mpc toggle`
+
+- Stop playing:
+
+`mpc stop`
+
+- Show information about currently playing song:
+
+`mpc status`
+
+- Play next song:
+
+`mpc next`
+
+- Play previous song:
+
+`mpc prev`
+
+- Seek playing song:
```

Comment:
I just found this:
https://en.wikipedia.org/wiki/Media_controls

And it calls it *"rewind"* so maybe it's actually right? Will try it in a new commit...